benchmarks/benchmarks/preprocessing.py

```diff
@@ -7,17 +7,22 @@
 
 from typing import TYPE_CHECKING
 
+from asv_runner.benchmarks.mark import skip_for_params
+
 import scanpy as sc
 from scanpy.preprocessing._utils import _get_mean_var
 
-from .utils import lung93k, pbmc3k, pbmc68k_reduced
+from .utils import bmmc4k, lung93k, pbmc3k, pbmc68k_reduced
 
 if TYPE_CHECKING:
     from anndata import AnnData
 
 
 adata: AnnData
 
+params = ["pbmc68k_reduced", "bmmc4k"]
```

Comment:
I am not sure this is what I realized what was happening.  I thought this was a PR for _very_ large datasets

Reply:
Ah I see there is a separate issue: https://github.com/scverse/scanpy/issues/3013